app/src/main/java/com/bigneon/doorperson/db/SyncController.kt

```diff
@@ -30,6 +31,10 @@ class SyncController {
         var ticketListItemPosition = -1
         var ticketListItemOffset = 0
 
+        var isNetworkAvailable: Boolean = false
+        var isSyncActive: Boolean = false
+        var isOfflineModeEnabled: Boolean = false
```

Comment:
Let's make the offline mode on the default
```suggestion
        var isOfflineModeEnabled: Boolean = true
```